var/spack/repos/builtin/packages/exodusii/package.py

```diff
@@ -27,11 +27,8 @@
 # TODO: Add support for a C++11 enabled installation that filters out the
 # TODO: "C++11-Disabled" flag (but only if the spec compiler supports C++11).
 
-# TODO: Add support for parallel installation that uses MPI.
-
-# TODO: Create installation options for NetCDF that support larger page size
-# TODO: suggested by Exodus (see the repository "README" file).
-
+# TODO: Use variant forwarding to forward the 'mpi' variant to the direct
+# TODO: dependencies 'hdf5' and 'netcdf'.
 
```

Comment:
There needs to be 2 blank lines here. That is what is causing the flake8 tests to fail.